scylla-server/src/processors/mqtt_processor.rs

```diff
@@ -66,16 +87,21 @@ impl MqttProcessor {
             .set_session_expiry_interval(Some(u32::MAX))
             .set_topic_alias_max(Some(600));
 
+        let rate_map: HashMap<String, Instant> = HashMap::new();
+
         // TODO mess with incoming message cap if db, etc. cannot keep up
```

Comment:
this basically what you're doing rn right?

Reply:
ya.

Reply:
go ahead and remove the todo unless you still need it for some reason